src/odemis/acq/align/z_localization.py

```diff
@@ -328,6 +323,180 @@ def _do_measure_z(f: ProgressiveFuture, stigmator, angle: float, pos: Tuple[floa
         f.result(timeout=600)
         f._task_state = FINISHED
 
+def estimate_measure_z(stigmator, angle: float, pos: Tuple[float, float], stream) -> float:
```

Comment:
the same function was already there previously from line 240-250. Is there a reason for this change?